instrumentation/kafka-clients/kafka-clients-0.11/library/src/main/java/io/opentelemetry/instrumentation/kafka/ClientSpanNameProvider.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.kafka;
+
+import java.util.function.BiFunction;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+/**
+ * Copied from OpenTracing.
+ *
+ * <p>Returns a string to be used as the name of the spans, based on the operation preformed and the
+ * record the span is based off of.
+ *
+ * @author Jordan J Lopez
```

Comment:
I think let's not pull in the `@author` tag, since we don't apply them anywhere else in the repo. And check out the example below for how to pull in the copyright:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d2794fa23185d6d3d5a773217714ae195f431cc8/instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/OpenTelemetryDriver.java#L1-L21